### PR TITLE
add an ide.host.json for nicer display in VS

### PR DESCRIPTION
**Pull request by baronfel**
## Proposed Changes

This adds some additional configuration that
* enables the VS New Project Dialog
* makes that New Project Dialog experience look nice

I got this information partially from @sayedihashimi's template validation dotnet tool: https://github.com/sayedihashimi/template-sample, which is a great tool.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] Build and tests pass locally
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

image
image

### CODE REVIEW

File: Content/.template.config/template.json [hunk 1]
```diff
@@ -2,25 +2,28 @@
     "$schema": "http://json.schemastore.org/template",
     "author": "Jimmy Byrd",
     "classifications": [
-        "Scaffold"
+        "Scaffold", "Console", "Library"
```
Comment (baronfel) [inline Content/.template.config/template.json:5]: These are tags that the VP new project dialog uses to filter templates

File: Content/.template.config/template.json [hunk 2]
```diff
@@ -2,25 +2,28 @@
...
     ],
+    "description": "A full-featured template for creating console or library solutions with documentation and release workflows.",
```
Comment (baronfel) [inline Content/.template.config/template.json:7]: These descriptions give nice tooltips, but they also help in the CLI tooltips/help as well.

File: Content/.template.config/template.json [hunk 3]
```diff
@@ -2,25 +2,28 @@
...
     "name": "MiniScaffold",
     "tags": {
-        "language": "F#"
+        "language": "F#",
+        "type": "solution"
```
Comment (baronfel) [inline Content/.template.config/template.json:11]: the `type` property allows this to show up in VS, not just in the CLI.

File: Content/.template.config/ide.host.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "http://json.schemastore.org/vs-2017.3.host",
+    "symbolInfo": [
+        {
+            "id": "githubUsername",
+            "name": {
+                "id": "githubUsername",
+                "text": "Github Username"
```
Comment (baronfel) [inline Content/.template.config/ide.host.json:8]: this is how the short names get replaced to longer user-friendly names in an IDE like VS.

**Comment by TheAngryByrd** [comment]
Thanks!

**Pull request merged**